### PR TITLE
Remove blind mobs reading ability

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
Blind people can no longer read.

## Why It's Good For The Game
Braille not included.

stream_free_preview

## Changelog
:cl:
fix: Remove blind mobs reading ability
/:cl:

### CODE REVIEW

File: code/game/machinery/dna_infuser/infuser_book.dm [hunk 1]
```diff
@@ -11,6 +11,14 @@
 	drop_sound = 'sound/items/handling/book_drop.ogg'
 	pickup_sound = 'sound/items/handling/book_pickup.ogg'
 
+/obj/item/infuser_book/ui_status(mob/user, /datum/ui_state/state)
+	if(user.is_blind())
+		to_chat(user, span_warning("You are blind and can't read anything!"))
```
Comment (MrMelbert) [inline code/game/machinery/dna_infuser/infuser_book.dm:16]: I'm not 100% sure that this should have to chat feedback in it

File: code/modules/antagonists/wizard/equipment/wizard_spellbook.dm [hunk 2]
```diff
@@ -136,6 +136,14 @@
 		possible_entry.set_spell_info() //loads up things for the entry that require checking spell instance.
 		entries |= possible_entry
 
+/obj/item/spellbook/ui_status(mob/user, /datum/ui_state/state)
+	if(user.is_blind())
+		to_chat(user, span_warning("You are blind and can't read anything!"))
+		return UI_CLOSE
```
Comment (MrMelbert) [inline code/modules/antagonists/wizard/equipment/wizard_spellbook.dm]: Please check that Dullahan Wizards can still read their spellbook, they're funny and I wanna keep them as a possible option.

**Review by MrMelbert — COMMENTED** [review]
You should make this a `/datum/ui_state`, ie `/datum/ui_state/physical_state/requires_sight` or whatever these items currently use.

Reduces copy pasta, makes it easier for future people to work with

File: code/modules/antagonists/wizard/equipment/wizard_spellbook.dm (hunk 2 above)
Comment (timothymtorres) [reply code/modules/antagonists/wizard/equipment/wizard_spellbook.dm]: Wa6W0FSavv

**Comment by Mothblocks** [comment]
We are in a feature freeze at the moment, feel free to remake this PR after January.

https://github.com/tgstation/tgstation/pull/80731

**Comment by timothymtorres** [comment]
Bugfix?

**Comment by timothymtorres** [comment]
@Mothblocks #80770

**Comment by Ghommie** [comment]
Welp, I guess you just have to remake the PR now. Mothblocks re-opened it and changed labels, only for you to delete the branch less than a minute after without noticing it.

**Comment by RunKittenzRComing** [comment]
I sincerely think a more creative approach would be awesome for this, given it's the year 2564, it wouldn't be outlandish to give them a roundstart skillchip or something that magically allows them to read through future-magic, similar to translator gloves. Going without it would revoke this ability. 

This PR as it stands would be incredibly debilitating to a _lot_ of jobs and people, and has no functional gain outside "realism". personally knowing a static blind main voice their opinion against this, I figure I'd just respectfully put my voice out there of an alternative fix that this PR could maybe be redone under.

**Comment by Ghommie** [comment]
The idea is nice and dandy, but unfortunately, we're on a feature freeze right now.

**Pull request closed without merging**